logger.js

```diff
@@ -133,6 +134,9 @@ function pinoLogger (opts, stream) {
       }
 
       if (shouldLogSuccess) {
+        const level = customLogLevel ? customLogLevel(res) : useLevel
```

Comment:
Could you add another check if `level` is undefined? In this case it should not log the next line.

Reply:
Fixed